drivers/ieee802154/ieee802154_stm32wba.c

```diff
@@ -0,0 +1,1075 @@
+/* ieee802154_stm32wba.c - STM32WBxx 802.15.4 driver */
+
+/*
+ * Copyright (c) 2025 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT st_stm32wba_ieee802154
+
+#define LOG_MODULE_NAME ieee802154_stm32wba
```

Comment:
These macros are not needed before the header files inclusion.
Could you move them after?